src/Surface.zig

```diff
@@ -272,6 +272,7 @@ const DerivedConfig = struct {
     title_report: bool,
     links: []Link,
     link_previews: configpkg.LinkPreviews,
+    scroll_to_bottom_on_keystroke: bool,
```

Comment:
We usually just store the full config struct. I think that's what we should do here, since we'll need it for future values.

Reply:
Done.